### PR TITLE
feat(components): Add test id in components 

### PR DESCRIPTION
**Pull request by CaioAugustoR**
[EUEG-2477](https://gympass.atlassian.net/browse/EUEG-2477)

## Description 📄

Update components on yoga with data test id

## Platforms 📲

- [x] Web
- [ ] Mobile

## Type of change 🔍

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [x] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

[Upload your screenshots here]

| Before | After |
| ------ | ----- |
|        |       |

[EUEG-2477]: https://gympass.atlassian.net/browse/EUEG-2477?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by caiotracera** [comment]
Hey, @CaioAugustoR! Congratulations on this PR! 🎉
Just a question: couldn't we pass the `data-testid` as a prop directly to the component? From what I saw, we are spreading the props on components, so I am afraid that this behavior is already implemented;

What do you think? Please, let me know if what I am saying make sense or not.

Cheers,
Caio 😁

File: packages/yoga/src/Accordion/web/Accordion.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { string, node, bool } from 'prop-types';
 import { ChevronDown } from '@gympass/yoga-icons';
 import { Text, Divider } from '@gympass/yoga';
 
-const Accordion = ({ title, subtitle, children, disabled, expanded }) => {
+const Accordion = ({ title, subtitle, children, disabled, expanded, testId }) => {
```
Comment (guilhermecardoso-gympass) [inline packages/yoga/src/Accordion/web/Accordion.jsx:7]: check this lintter sugestion

**Comment by CaioAugustoR** [comment]
> 

Hey Caio,

It's a great idea, I'm going to make some changes so we don't have to send another props

**Review by MicaelRodrigues — COMMENTED** [review]
Just a thought for discussion.
We should avoid facilitate testing by `test-id`...Users don't use 'test-ids' to use their components.

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-the-wrong-query
https://betterprogramming.pub/why-you-should-avoid-testing-react-components-with-test-ids-ee50d20d37d2

**Review by eduardobreno — APPROVED** [review]

**Review by MicaelRodrigues — APPROVED** [review]

**Review by thiamancio — APPROVED** [review]

**Review by guilhermecardoso-gympass — APPROVED** [review]

**Pull request merged**